docs/workflows/create_sync_publish.rst

```diff
@@ -59,6 +59,31 @@ Remote GET response:
 
 .. _versioned-repo-created:
 
+Suse Linux Enterprise Server Remote
+***********************************
+
+If you would like to sync SLES 12+ repository you will need to specify an authentication as ``sles_auth_token``.
```

Comment:
Just for my understanding, before SLES 12 there were no auth token required ever, and now for SLES 12+ it's required always, correct?

Reply:
yes for SLES 12+ **official** repositories auth token is always required.

Reply:
SLES11 repos requires basic authentication via username and password, but since SLES12 it has changed to an auth token.